src/typescript/frontend/src/components/loading.tsx

```diff
@@ -4,19 +4,36 @@
 import React, { useEffect, useMemo } from "react";
 import AnimatedStatusIndicator from "./pages/launch-emojicoin/animated-emoji-circle";
 import { getRandomSymbolEmoji, SYMBOL_EMOJI_DATA, type SymbolEmojiData } from "@sdk/emoji_data";
+import { Emoji } from "utils/emoji";
 import { usePathname } from "next/navigation";
 import { EMOJI_PATH_INTRA_SEGMENT_DELIMITER, ONE_SPACE } from "utils/pathname-helpers";
+import { type EmojiMartData } from "./pages/emoji-picker/types";
+import { init } from "emoji-mart";
 
 const unpathify = (pathEmojiName: string) =>
   SYMBOL_EMOJI_DATA.byName(pathEmojiName.replaceAll(EMOJI_PATH_INTRA_SEGMENT_DELIMITER, ONE_SPACE));
 
+const data = fetch("https://cdn.jsdelivr.net/npm/@emoji-mart/data@latest/sets/15/native.json").then(
+  (res) =>
+    res.json().then((data) => {
+      return data as EmojiMartData;
+    })
+);
+
 export const Loading = ({
   emojis,
   numEmojis,
 }: {
   emojis?: SymbolEmojiData[];
   numEmojis?: number;
 }) => {
+  useEffect(() => {
+    data.then((d) => {
+      init({ set: "native", data: d });
+    });
```

Comment:
```suggestion
  // Fetch/load the emoji picker data here to ensure that the picker has emoji data to use.
  // Since the library only initializes when the picker component is rendered, the library won't have
  // data on pages that don't use the picker component unless we explicitly call `init(...)` here.
  useEffect(() => {
    data.then((d) => {
      init({ set: "native", data: d });
    });
```